students/yujeong/users/views.py

```diff
@@ -33,4 +33,21 @@ def post(self, request):
             return JsonResponse({"message": "SUCCESS"}, status = 201)
             
         except KeyError :
-            return JsonResponse({"message": "KEY_ERROR"}, status = 400)       
\ No newline at end of file
+            return JsonResponse({"message": "KEY_ERROR"}, status = 400)
+
+class SignInView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try: 
+            if not User.objects.filter(email=data['email']).exists() :
+                return JsonResponse({"message": "INVALID_USER"}, status = 401)   
+
+            if User.objects.filter(email=data['email']):
+                if User.objects.filter(password=data['password']):
+                    return JsonResponse({"message": "SUCCESS"}, status = 200) 
+                else:
+                    return JsonResponse({"message": "INVALID_USER"}, status = 401) 
```

Comment:
filter 안에 조건을 1개가 아닌 여러개를 넣어서 판단 할 수 있습니다.
만약 아래와 같이 코드를 수정한다면 email과 password를 동시에 만족하는 user를 찾기 때문에 위에 email을 판단하는 코든가 필요 없을 수도 있습니다.
그러나 인증 인가를 적용하면 비밀번호를 검증하는 부분을 따로 해야 되기 때문에 위의 email 검증 코드는 일단 놔두겠습니다.
```suggestion
            if not User.objects.filter(email=data['email'], password=data['password']):
                return JsonResponse({"message": "INVALID_USER"}, status = 401)
            
             return JsonResponse({"message": "SUCCESS"}, status = 200) 
```